setup.cfg

```diff
@@ -73,17 +73,11 @@ docs =
     sphinxcontrib-serializinghtml==1.1.5
 
 [mypy]
+strict = True
 ignore_missing_imports = True
 strict_optional = True
 show_error_codes = True
 
-# strictness
-strict = True
-# these options reduce the strictness and should eventually be removed
```

Comment:
This is an outdated comment since all these options have been set to True. At this point they are superfluous since they are implied by `strict=True`.